bears/php/PHPMessDetectorBear.py

```diff
@@ -0,0 +1,38 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import \
+    DistributionRequirement
+
+
+@linter(executable='phpmd',
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>\d+)(?P<message>.*)')
```

Comment:
Will update my regex to `(?P<filename>.*):(?P<line>\d+)\s*(?P<message>.*)`, note that I've added an `\s*` to capture spaces. :)